src/core/security/credentials.c

```diff
@@ -180,7 +180,7 @@ void grpc_server_credentials_set_auth_metadata_processor(
       "grpc_server_credentials_set_auth_metadata_processor("
       "creds=%p, "
       "processor=grpc_auth_metadata_processor { process: %lx, state: %p })",
-      3, (creds, (unsigned long)processor.process, processor.state));
```

Comment:
I will break out the core changes I need to Bigtable.NET from x64 compilation issues.